_includes/features.html

```diff
@@ -0,0 +1,100 @@
+                    <p class="feature-text">JanusGraph is a transactional database that can support thousands of concurrent users executing complex graph traversals in real time. Support for ACID and eventual consistency.</p>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="feature-icon fas fa-database"></i> &nbsp; Data Storage
+                </p>
+                <div>
+                    <p class="feature-text">Graph data can be stored in:</p>
+                    <ul class="central-list">
+                        <li>Apache Cassandra</li>
+                        <li>Apache HBase</li>
+                        <li>Google Cloud Bigtable</li>
+                        <li>Oracle BerkeleyDB</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="feature-icon fas fa-search"></i> &nbsp; Search
+                </p>
+                <div>
+                    <p class="feature-text">Secondary indexes are supported via:</p>
+                    <ul class="central-list">
+                        <li>Elasticsearch</li>
+                        <li>Apache Solr</li>
+                        <li>Apache Lucene</li>
+                    </ul>
+
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="fas fa-chart-pie"></i> &nbsp; Analytics
+                </p>
+                <div>
+                    <p class="feature-text">JanusGraph OLAP integrations include:</p>
+                    <ul class="central-list">
+                        <li>Apache Spark</li>
+                        <li>Apache Hadoop</li>
+                        <li>Apache Giraph</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="fas fa-tasks"></i> &nbsp; TinkerPop
+                </p>
+                <div>
+                    <p class="feature-text">Native integration with the Apache TinkerPop™ graph stack:</p>
+                    <ul class="central-list">
+                        <li>Gremlin graph query language</li>
+                        <li>Gremlin graph server</li>
+                        <li>Gremlin applications</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="col-sm-4 portfolio-item feature-item">
+                <p class="second-title"><i class="fas fa-plug"></i> &nbsp; Adapters
+                </p>
+                <div>
+                    <p class="feature-text">There are different 3rd party storage adapters for JanusGraph:</p>
+                    <ul class="central-list">
+                        <li>Couchbase</li>
```

Comment:
If we mention these 3rd party adapters here, shouldn't we also link to them?

Reply:
Sorry about Couchbase. I thought there is an adapted but there is no yet Couchbase adapter for JanusGraph. I will provide links to FoundationDB and DynamoDB adapters.